src/tests/users/signup.test.js

```diff
@@ -1,30 +1,268 @@
 const expect = require('expect')
 const { request } = require('../../utils/test')
-const { UserFactory } = require('../../utils/factories/')
+const { UserFactory, QualificationFactory } = require('../../utils/factories/')
+const Qualification = require('../../models/qualifications')
+const User = require('../../models/user')
 
 describe('signup', () => {
   const fakeUser = UserFactory.generate()
+  const fakeUser2 = UserFactory.generate()
+  let users = []
+  let skills = []
+  let knowledge = []
+  let user = null
+  before(async () => {
+    for (let y = 0; y < 5; y++) {
+      const skill = await Qualification.create(
+        QualificationFactory.generateSkill()
+      )
+      skills.push(`"${skill._id}"`)
+      const knowledge1 = await Qualification.create(
+        QualificationFactory.generateKnowledge()
+      )
+      knowledge.push(`"${knowledge1._id}"`)
+    }
+    for (let x = 0; x <= 1; x++) {
+      let user = await UserFactory.generate()
+      user = await User.signup(user)
+      users.push(user)
+    }
+  })
+
+  it("should not signup if mentor is not certified or mentor's email is not verified", () => {
```

Comment:
Your new tests should be appended - not prepended. Your new tests should be added at the end of the file

Reply:
this should be added at top